python/tvm/relax/transform/legalize_ops/statistical.py

```diff
@@ -45,7 +45,7 @@ def _te_mean(x: te.Tensor, axis: List[tir.IntImm], keepdims: bool) -> te.Tensor:
 
 
 def _te_variance(x: te.Tensor, axis: List[tir.IntImm], keepdims: bool) -> te.Tensor:
-    dev = x - _te_mean(x, axis, keepdims)
```

Comment:
https://github.com/mlc-ai/relax/pull/168

Reply:
consider send to unity